trie/encoding.go

```diff
@@ -53,7 +53,7 @@ func hexToCompact(hex []byte) []byte {
 
 // hexToCompactInPlace places the compact key in input buffer, returning the length
 // needed for the representation
```

Comment:
Pls fix the comment "returning the length [...]" -> "returning the compacted key".